src/types.ts

```diff
@@ -85,6 +87,7 @@ export interface VMDisk {
             port: optString;
         },
         startupPolicy: optString;
+        device?: optString; // XXX - never set?
```

Comment:
`#include <pitti/standard_XXX_complaint.h>`

Reply:
Yep, we are cleaning up, so let's clean that up! :-)

I will remove the bogus entry from DISK_SOURCE_LIST instead.